flatpak-repos/eos-sdk.flatpakrepo

```diff
@@ -0,0 +1,9 @@
+[Flatpak Repo]
+Title=Endless Apps Runtimes
+Url=https://ostree.endlessm.com/ostree/eos-sdk
+Homepage=https://endlessm.github.io/eos-knowledge-lib/
+Comment=Flatpak runtime for developing apps for Endless OS
+Description=Endless apps runtimes are released on a three month schedule and contain the libraries necessary for developing and running offline content apps. They receive minor bug fixing and security updates, and should be considered ABI stable and frozen.
```

Comment:
I know this is just cut and paste from what's already there, but I probably would not state about the three month schedule in the description, as that is an implementation detail that is subject to change.

Reply:
Fixed in new PR.

Reply:
Fixed but can't find where the file was originally defined - I assume it was copied out of somewhere?

Reply:
It's in the root dir of eos-knowledge-lib, I think.